tests/test_command_reference.rb

```diff
@@ -45,13 +45,15 @@ def write(string)
 
   def test_data_sanity
     # Make sure the actual YAML in our library loads for various platforms
-    CommandReference.new('', '')
-    CommandReference.new('NXAPI', '')
-    CommandReference.new('NXAPI', 'N9K-C9396PX')
-    CommandReference.new('NXAPI', 'N7K-C7010')
-    CommandReference.new('NXAPI', 'N3K-C3064PQ-10GE')
-    CommandReference.new('gRPC', '')
-    CommandReference.new('gRPC', 'R-IOSXRV9000-CH')
+    CommandReference.new
+    CommandReference.new(platform: :nexus, cli: true)
```

Comment:
My reason for defaulting cli (and any future API types) to false is that this makes it easier to add additional API types with minimal impact. Additionally it forces us to explicitly specify which APIs are supported by a given client - think:

```ruby
CommandReference.new(platform: :ios_xe, relax_ng: true)
CommandReference.new(platform: :ios_xr, yang: true, cli: true)
CommandReference.new(platform: :nexus, cli: true)
```

vs:

```ruby
CommandReference.new(platform: :ios_xe, relax_ng: true, cli:false, yang: false)
CommandReference.new(platform: :ios_xr, yang: true, relax_ng: false) # cli: true is default so we omit it
CommandReference.new(platform: :nexus, cli: true, yang: false, relax_ng: false)
```